include/no_os_print_log.h

```diff
@@ -42,6 +42,8 @@
 
 #include <stdio.h>
 
+//#define PRINT_TIME
```

Comment:
this shouldnt be here

Reply:
I removed it. It will have to be defined in application configuration files.